public/css/team.css

```diff
@@ -1,70 +1,83 @@
-.header {
-	visibility: hidden;
-	margin: 0px;
-	width: 0;
+/* Header */
+.logo-header {
+	display: flex;
+	top: -85px;
+	align-items: center;
+	justify-content: center;
+}
+
+.logo-img-header {
+	position: absolute;
+	top: 170px;
 }
 
-.directors-header {
-	margin: 50px 0px 40px 0px;
-	display: block;
-	max-width: 65%;
-	display: inline-block;
+.logo-img {
+	margin-left: 50%;
+	transform: translateX(-50%);
+	max-width: 100%;
+}
+
+.logo-txt {
+	display: inline;
+	color: white;
+	position: absolute;
+	margin-left: 50%;
+	transform: translateX(-50%);
+	top: 70px;
+	left: 120px;
+	right: -80px;
+	font-size: 34px;
+	font-weight: bold;
 }
 
 .img-header {
 	width: 100%;
+	min-width: 800px;
 }
 
-.team {
-	padding-bottom: 40px;
+/* General */
+
+.filler-thick,
+.filler {
+	width: 100%;
+	height: 30px;
 }
 
-.team-container {
-	text-align: center;
-	padding-bottom: 100px;
-	background-color: #fbf3f0;
+.filler-thick {
+	height: 50px;
 }
 
+/* text / photo */
+
 p {
-	color: white;
+	color: var(--green);
 	font-size: 20px;
 	text-align: center;
 	line-height: 24px;
 	margin-bottom: 45px;
 }
 
-.Title {
-	margin: 130px 0px 20px 0px;
-	display: block;
-	max-height: 45px;
-	display: inline-block;
-}
-
-.Description,
-.lead {
-	color: #2b2b2b;
-	font-family: "Roboto", sans-serif;
+.Description {
+	color: var(--green);
+	font-family: "Roboto Condensed", sans-serif;
```

Comment:
Not 100% sure, so you might want to check with Robert or someone else, but aren't we using regular "Roboto" for the page font and not "Roboto Condensed"? It's hard to tell the difference between the two, but the condensed version looks like it has slightly thinner letters.

Reply:
On Figma, the font is labeled as "Roboto Condensed" (at least on the team page). Although I agree, it's pretty hard to tell between the two.

Reply:
I know this was a while ago and I already made an announcement about it since this thread was started, but to reiterate, Roboto Condensed was a mistake by the design committee. You can just use Roboto.